src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.Arrays;
+
+/**
+ * A class that extends the Task class, containing information about deadlines.
+ */
+public class Deadline extends Task {
+
+    private LocalDate date;
+
+    public Deadline(String description, String date) {
```

Comment:
This may feel a bit cluttered, you could consider making a separate method to process the Date